source/mongodump.txt

```diff
@@ -302,8 +302,33 @@ value overrides the read preference specified in the URI string.
 Data Exclusion
 ~~~~~~~~~~~~~~
 
-:binary:`~bin.mongodump` excludes the content of the ``local`` database
-in its output.
+If you specify both the :option:`--db <mongodump --db>` and the
+:option:`--collection <mongodump --collection>` options, ``mongodump``
+dumps the specified collection.
+
+If you only specify the :option:`--db <mongodump --db>` option,
+``mongodump`` dumps that database with the following exclusions:
+
+- If the value of ``--db`` is ``admin``, ``mongodump`` excludes the
+  ``system.keys`` collection. 
```

Comment:
Suggest splitting this into behavior when user is/isn't connected to an Atlas proxy:

```suggestion
- If the value of ``--db`` is ``admin``:
  - If you aren't connected to an Atlas Proxy, ``mongodump`` dumps the database but 
    excludes the ``system.keys`` collection. 
  - If you're connected to an Atlas Proxy, ``mongodump`` attempts to dump the database 
    but errors out.
```